frontends/ast/genrtlil.cc

```diff
@@ -771,8 +771,6 @@ struct AST_INTERNAL::ProcessGenerator
 					if (node->type == AST_CONSTANT && node->is_string) {
 						arg.type = VerilogFmtArg::STRING;
 						arg.str = node->bitsAsConst().decode_string();
-						// and in case this will be used as an argument...
```

Comment:
Shouldn't an equivalent change be made in the other STRING emitting code in `AstNode::processFormat` in `simplify.cc`?